modules/ocm-nw-networkpolicy-create.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * networking/network_policy/creating-network-policy.adoc
+// * post_installation_configuration/network-configuration.adoc
+
+:name: network
+:role: admin
+ifeval::["{context}" == "configuring-multi-network-policy"]
+:multi:
+:name: multi-network
+:role: cluster-admin
+endif::[]
+
+:_content-type: PROCEDURE
+[id="ocm-nw-networkpolicy-create_{context}"]
+= Creating a {name} policy using {cluster-manager}
+
+To define granular rules describing ingress or egress network traffic allowed for namespaces in your cluster, you can create a {name} policy.
+
+.Prerequisites
+
+* You logged in to {cluster-manager-url}.
+* You created an {product-title} cluster.
+* You configured an identity provider for your cluster.
+* You added your user account to the configured identity provider.
+
+.Procedure
+
+. From {cluster-manager-url}, click on the cluster you want to access.
+
+. Click *Open console*.
+
+. Click on your identity provider and provide your credentials to log into the cluster.
+
+. Under *Networking*, click *NetworkPolicies*.
+
+. Click *Create NetworkPolicy*.
+
+. Provide a name for the policy in the "Policy name" field.
```

Comment:
I believe field names are bold as well, unless we've adopted a different style for OSD.